packages/manager/apps/pci-databases-analytics/src/lib/durationHelper.spec.tsx

```diff
@@ -87,3 +88,39 @@ test('durationStringToHuman', () => {
     '2 años, 5 meses, 3 días, 4 horas, 50 minutos, 45 segundos',
   );
 });
+
+
+describe('durationStringToSeconds', () => {
+  it('should return 0 for an empty string', () => {
```

Comment:
An it.each could be more concise on your code

Reply:
Yes, you're right, fixed. Thanks